tests/infra/proc.py

```diff
@@ -7,9 +7,11 @@
 from loguru import logger as LOG
 
 
-def ccall(*args, log_output=True):
-    LOG.info(" ".join(args))
-    result = run(args, capture_output=True)
+def ccall(*args, path=None, log_output=True):
+    suffix = f" [cwd: {path}]" if path else ""
```

Comment:
f" [cwd: {path or os.getcwd()}]" maybe?

Reply:
I thought it'd be better to keep the test output as concise as possible. If the command is run in the current working directory, then anyone trying to reproduce the test manually should be able to copy and execute that command verbatim.